src/PresentationalComponents/Common/Tables.js

```diff
@@ -0,0 +1,28 @@
+// Builds returns url params from table filters, pushes to url if history object is passed
+export const urlBuilder = (filters, history) => {
+    const queryString = `?${Object.keys(filters).map(key => `${key}=${Array.isArray(filters[key]) ? filters[key].join() : filters[key]}`).join('&')}`;
+    history && history.replace({
+        search: queryString
+    });
+    return queryString;
+};
+
+// transforms array of strings -> comma seperated strings, required by advisor api
+export const filterFetchBuilder = (filters) => (Object.assign({},
+    ...Object.entries(filters).map((filter) => {
```

Comment:
You can use array destructing instead of declaring the constants :wink: 
```jsx
...Object.entries(filters).map(([filterName, filterValue]) => {
  ...
```

Reply:
awesome catch!